builds/e2e/longhaul.yaml

```diff
@@ -285,11 +317,27 @@ jobs:
         fetchDepth: 100
         submodules: recursive
       - task: AzureKeyVault@1
-        displayName: 'Azure Key Vault'
+        displayName: 'Common Test Azure Key Vault'
+        inputs:
+          azureSubscription: $(azure.subscription)
+          KeyVaultName: 'edgebuildkv'
+          SecretsFilter: >- 
+            edgebuilds-azurecr-io-username,
+            edgebuilds-azurecr-io-pwd,
+            StorageAccountMasterKeyStress,
+            SnitchLongHaulAlertUrl,
```

Comment:
Ideally we would remove this, but I think it is better to have a separate scoped PR for removing the old longhaul logic (containing lingering refs to snitcher and analyzer).